.github/ISSUE_TEMPLATE/bug-report-template.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Notify Bug Template
+about: Create a report to help us improve
+title: 'BUG: '
+labels: Bug, Notify
+assignees: ''
+
+---
+
+<!--
+Remove any comment indicators like the one above or below if you want to include a section. 
+We do not need what to write showing up in tickets though, so please do not uncomment those descriptions.
+-->
+
+## Description
+
+- [ ] Ticket is understood, and QA has been contacted (if the ticket has a QA label).
+
+<!--_**Required.** Describe the problem._-->
+
+## Steps to Reproduce
+<!--_**Required.** Provide information on what steps you are aware of that produce this undesired outcome.._-->
+
+1. 
+
+<!--
+## Workaround
+Is there something we can do to work around this issue in the meantime?
+-->
+
+## Impact/Urgency
+<!--_**Required.** Describe the impact this bug has on our system, clients, and/or team._-->
+
+## Expected Behavior
+<!--_**Required.** Describe the desired outcome if this were functioning as expected. Include a checklist if applicable._-->
+
+- [ ] 
+
+<!--
+## QA Considerations
+_For QA to populate. Leave blank if QA is not applicable on this ticket._
```

Comment:
```suggestion
_Populate with suggested QA actions if applicable. Leave blank if QA is not applicable on this ticket._
```

Could we replace this comment on both templates please?

Reply:
Updated